### PR TITLE
Interactive tool for higlass

### PR DESCRIPTION
**Pull request by joachimwolff**

### CODE REVIEW

File: tools/interactive/interactivetool_higlass.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<tool id="interactive_tool_higlass" tool_type="interactive" name="HiGlass" version="1.8.0">
+    <requirements>
+        <container type="docker">higlass/higlass-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="HiGlass Visualisation on $infile.display_name" requires_domain="True">
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml]: this variable does not exists, it needs to be matrix

File: tools/interactive/interactivetool_higlass.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<tool id="interactive_tool_higlass" tool_type="interactive" name="HiGlass" version="1.8.0">
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml:1]: maybe add a description, that HiGlass is a visualisation for HiC-cool data

File: tools/interactive/interactivetool_higlass.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <entry_points>
+        <entry_point name="HiGlass Visualisation on $infile.display_name" requires_domain="True">
+            <port>80</port>
+        </entry_point>
+    </entry_points>
+    <command><![CDATA[
+        mkdir higlass-local-data
+        && mkdir higlass-local-tmp
+        
+        && ln -s '$matrix' higlass-local-tmp/'matrix_name.$matrix.ext'
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml]:
```suggestion
        && ln -s '$matrix' 'higlass-local-tmp/matrix_name.$matrix.ext'
```

File: tools/interactive/interactivetool_higlass.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+         
+        && python higlass-server/manage.py ingest_tileset --filename /tmp/'matrix_name.$matrix.ext' --filetype cooler --datatype matrix
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml]: see the quoting above ...

File: tools/interactive/interactivetool_higlass.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        
+    ]]>
+    </command>
+    <inputs>
+        <param name="matrix" type="data" format="cool" label="Cool file with multiple resolutions"/>
+    </inputs>
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml:23]: there needs to be an output (for now)

File: tools/interactive/interactivetool_higlass.xml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <tests>
+    </tests>
+    <help>
+    Interactive tool for visualising Hi-C data
```
Comment (bgruening) [inline tools/interactive/interactivetool_higlass.xml]: Can you please add some more description. About the filetype restrictions for example.

Please also add the citation.

**Review by bgruening — APPROVED** [review]
@joachimwolff a review is inline.

This PR needs to go against: https://github.com/usegalaxy-eu/galaxy

File: tools/interactive/interactivetool_higlass.xml (hunk 5 above)
Comment (joachimwolff) [reply tools/interactive/interactivetool_higlass.xml:23]: There is none, so just an empty file?

Comment (bgruening) [reply tools/interactive/interactivetool_higlass.xml:23]: Yes. We just need this to indicate the status for the user atm.

**Pull request closed without merging**